im2latex/conf/config.yaml

```diff
@@ -0,0 +1,51 @@
+mode: "train"
+log_level: 1
+
+torch:
+  float32_matmul_precision: "high"
+  seed: 42
+
+dataset:
+  train_dataset_path: "OleehyO/latex-formulas"
+  split_dataset_name: "cleaned_formulas"
+  val_test_size: 0.2
+
+model:
+  tokenizer_name: "gpt2"
+  feature_extractor: "microsoft/swin-base-patch4-window7-224-in22k"
+  encoder_name: "microsoft/swin-base-patch4-window7-224-in22k"
+  decoder_name: "gpt2"
+  max_length: 512
+
+# Not part of original configs but was hardcoded in the original code
+decoding:
+  max_length: 256
+  early_stopping: True
+  no_repeat_ngram_size: 3
+  length_penalty: 2.0
+  num_beams: 4
+
+training:
+  distributed: False
+  num_epochs: 10
+  batch_size_train: 32
+  batch_size_val: 32

```

Comment:
So a batch size of 8 takes 14GB on my 4080. 12 should take 21GB on a 4090. A batch size of 20 should work on an A100 40GB. Maybe we should make this dynamic somehow (It's a minor problem)